graphviz2drawio/models/SvgParser.py

```
@@ -15,6 +15,7 @@
 from .commented_tree_builder import COMMENT, CommentedTreeBuilder
 from .CoordsTranslate import CoordsTranslate
 from .Errors import MissingTitleError
+import xml.etree.ElementTree as ET
```

Ruff (0.8.0)

`xml.etree.ElementTree` imported but unused

Remove unused import: `xml.etree.ElementTree`

(F401)